src/components/Literals/RemoteLiteralMapViewer.tsx

```diff
@@ -38,7 +38,7 @@ export const RemoteLiteralMapViewer: React.FC<{
         );
     }
 
-    if (map !== undefined) {
+    if (map !== undefined && map !== null) {
```

Comment:
If you're handling both cases, a coercing equality against `null` will do it and is a common pattern in this repository. 
`if (map != null) { ... }`

Generally when checking protobuf fields which are not primitive values, I use this form of check.